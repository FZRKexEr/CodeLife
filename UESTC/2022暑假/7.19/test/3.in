10
okidxibhkeecuxmkaawceujlrwongjojbdflowfgodouyjvrxhxpcujzmpwhqqxhewiyoeoeurzhwoekzdcjuxvtdmwzxqadapcg
1
pnxjigohwwallubiaxghzhdszfwqarxzajbwaspsseoishbncrfablkaevsxghaujzfsnsvtaizekvisqycpvcenaxlrspekhdrx
9
kmiimahxophatjvomvexjhiofggncafkeqcqypffazbalvmrtmvmryivevknwgwvzvwvropukkcfyjzgzremabvinthfpfgcoics
9
qgzojmesripepapytcfltgtuwwbfqhflptdlqubdpyiesuttnqrisnvohfxjkqdbfbntvaqowtrzlomfixbalogmqctkzsezmbpm
6
wsuzxjkfpixlbgijeqahuqwzkblluoqwwxauxftjinyoincxbxpibltvjiwcnwjicjejbyojhevgwltjnemlswcnhqknlpumgucq
7
ntnkffwitflrirqwhbervxkpyzuumwhicpaholrnhoiixehkxmzyuswxjtomgnfvqoegrkqpfvmakxajxdpkszrximcqdnqsqpbb
1
apnmdctrraicphtbncaiehtmdgspsdrymjkfeiqbbnepuikkcvuiaptymldkunqkwbilaskrrvpnhowveafbhdnrqodwtidonxnk
5
kmdfwyinyxaciujwhxsnydcferccwlqmoewsgrkkzfkpslmpelnmyfupdqnmuunxcyxclfyfetdqbijbwpcbanzvsboaroxapkol
6
gfdfllfbjvggvsynhfuwcdqwkvjsjsfpotmanocatlekuitafygxcmnaibnbpfhbxiuytskujdrtyzvjqqcumjoeknyawfgffffx
9
bzpjpolzijlpibincumejcyysbqtqdrtgemoksmqcsxzcbpvxenikfgnskatkxvdrlxgnjbrdqbcbuljboffdapwcsxracmgnlai
5
