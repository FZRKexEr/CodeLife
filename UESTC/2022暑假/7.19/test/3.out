882499718
2048
2048
65536
16384
882499718
1048576
65536
2048
1048576
